main.py

```diff
@@ -0,0 +1,13 @@
+from bot import Aria
+from os import environ
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+# .envファイルからロード
+load_dotenv()
+
+bot = Aria()
+
+extensions = []
+
+bot.run(environ.get('BOT_TOKEN'))
```

Comment:
トークンを取得する段階で何かしらのエラーを出すようにしたい。
```suggestion
token = environ['BOT_TOKEN']
bot.run(token)
```

Reply:
Whichever seems okay for me. Do you have any reason for preference on this correction?